modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1908,6 +1908,43 @@ public void testGetObjectEntriesAggregationFacets() throws Exception {
 		Assert.assertEquals(facetValue.getNumberOfOccurrences(), (Integer)2);
 	}
 
+	@Test
+	public void testGetObjectEntriesAggregationManyToManyRelationship()
+		throws Exception {
+
+		ObjectDefinition objectDefinition = _createObjectDefinition(
+			Arrays.asList(
+				new TextObjectFieldBuilder(
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).name(
+					"textObjectFieldName"
+				).build()));
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipLocalService.addObjectRelationship(
+				_adminUser.getUserId(),
+				objectDefinition.getObjectDefinitionId(),
+				objectDefinition.getObjectDefinitionId(), 0,
+				ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
+				LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+				"manyToManyRelationshipName",
+				ObjectRelationshipConstants.TYPE_MANY_TO_MANY);
```

Comment:
@pedro-oliveira446 can we use an existing object definition? `_objectDefinition1` for example.